.env.example

```
@@ -1,34 +1,17 @@
+L9_STATE_DB_PATH=./state.db
+L9_REQUIRE_SIGNATURE=false
+L9_SIGNING_SECRET=
+L9_SIGNING_ALGORITHM=hmac-sha256
 L9_JSON_LOGS=true
-L9_BOOTSTRAP_RUNTIME=true
+L9_LOG_LEVEL=INFO
+L9_MAX_PACKET_BYTES=1048576
+L9_MAX_HOP_DEPTH=16
+L9_ALLOWED_CLOCK_SKEW_SECONDS=30
+L9_CONTRACTS_DIR=./contracts
+APP_MODULE=engine.main:app
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_MODULE key should go before the L9_ALLOWED_ACTIONS key

(UnorderedKey)